blocklib_generator/CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+cmake_minimum_required(VERSION 3.28)
+project(MyProject LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 23)
+
+option(ENABLE_SPLIT "Enable splitting of generated files (pass --split to the parser)" ON)
```

Comment:
Need a better name for `ENABLE_SPLIT` not clear without the CMake file context.

Reply:
Ideas? I can not think of anything that is not really verbose.

Reply:
What about:

```cmake
option(ENABLE_PARALLEL_BLOCK_INSTANTIATIONS 
"Split specific block-type template instantiation into multiple files. Improves parallel compilation, reducing overall build time" 
ON)
```